File: src/main/java/ru/dkolmogortsev/controls/TimeEntryButton.java
package ru.dkolmogortsev.controls;

import javafx.geometry.Pos;
import javafx.scene.control.Button;

/**
 * Created by dkolmogortsev on 2/26/17.
 */
public class TimeEntryButton extends Button
{

    public TimeEntryButton()
    {
        super();
        setMaxSize(Double.MAX_VALUE, Double.MAX_VALUE);
        getStyleClass().clear();
        setAlignment(Pos.CENTER);
    }
}

File: src/main/kotlin/ru/dkolmogortsev/controls/TimeEntryButton.kt
package ru.dkolmogortsev.controls

import javafx.geometry.Pos
import javafx.scene.control.Button

/**
 * Created by dkolmogortsev on 2/26/17.
 */
class TimeEntryButton : Button() {
    init {
        setMaxSize(java.lang.Double.MAX_VALUE, java.lang.Double.MAX_VALUE)
        styleClass.clear()
        alignment = Pos.CENTER
    }
}
